main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
```

Comment:
Unfortunately, this doesn't actually comment out code. Replace // with # in order to do so!